CMakeLists.txt

```diff
@@ -54,7 +56,7 @@ if(enable-wayland)
     if (Qt6_FOUND)
       find_package(Qt6 REQUIRED COMPONENTS WaylandClient WaylandGlobalPrivate)
     else()
-      find_package(Qt5 5.14 REQUIRED COMPONENTS WaylandClient XkbCommonSupport)
+      find_package(Qt5 ${QT_MIN_VERSION} REQUIRED COMPONENTS WaylandClient XkbCommonSupport)
```

Comment:
The changes in this file seem unnecessary? The define for the Qt6 case is not used anywhere, and the Qt5 one is only used here, where there won't be any newer minor releases.

Reply:
I dropped this whole diff to `CMakeLists.txt` to get this PR applying on top of #125 and everything builds on my end anyway.